tests/ti_deps/deps/test_trigger_rule_dep.py

```diff
@@ -19,6 +19,8 @@
 import unittest
 from datetime import datetime
 
+import mock
```

Comment:
```suggestion
from unittest.mock import Mock
```

and change the usage. (Py3 includes mock, we should move away from using a the separate module as it's not needed)